src/views/AboutMonarch.md

```diff
@@ -1,137 +1,82 @@
-
-##### Analysis
-
-
-<figure class="right" href="#fig4">
-  <img src="../assets/img/pheno_species_coverage.jpg" height="200px">
-  <figcaption>
-
-Fig 4. Availability of curated phenotypes (black) attributed to human genes (direct or inferred via orthology) across a range of model organisms (Hs, Homo sapiens; Mm, Mus musculus; Rn, Rattus norvegius; Dr, Danio rerio; Ce, Ceanorabditis elegans). Merge shows the overall representation of phenotypes to unique human genes, where at least one organism contriubtes phenotypes. Model organisms may account for at least half of the overall phenotypic knowledge to understand human gene function. (Data as of March 2014.)
-
-  </figcaption>
-</figure>
-
-
-Given the integrated data sources, and their semantic mappings, we can start to do broad analysis of the data landscape. For example, we have found that while &lt;40% of human genes have been linked to phenotypes either directly, via GWAS studies, or inferred through disease associations, there is &gt;75% coverage (via orthology) when you expand to the five most-typically studied model organisms ([Fig 4](#fig4)). As we expand our database to include other organisms, this is sure to rise.
-
-We can also confirm what we have known for a long time, which is that different model organisms are used to study different areas of biology.
-
-
-#### Collaborations and Partners
-
-We are proud to work with several organizations to develop ontologies, drive community standards, research rare disease, and influence publishers and the research community to make research results more identifiable and reproducible, including:
+<h2 class="page-title">The Monarch Initiative</h2>
+<div class="row">
 
+<p>The Monarch Initiative is an integrative data and analytic platform connecting phenotypes to genotypes across 
+species, bridging basic and applied research with semantics-based analysis. The correlation of phenotypic outcomes 
+and disease with genetic variation and environmental factors is a core pursuit in biology and biomedicine.</p>
+ 
+</div>
+<div class="row">
+    <div class="col-lg-6">
+        <h5 class="figure-title">The phenotype annotation coverage of human coding genes.</h4>
+        <figure class="cross-species">
+          <img src="../assets/img/cross-species.jpeg" style="max-height:300px;" />
```

Comment:
This file needs to be updated.
Replace with file phenotype-cov-hum-5.png - found in the same directory (/assets/img/).

Reply:
update to phenotype-cov-hum-6.png